apps/academy/src/pages/tracks/nft-solidity/2.mdx

```diff
@@ -927,7 +925,7 @@ require("@nomicfoundation/hardhat-toolbox");
 
 const WALLET_PRIVATE_KEY = "YOUR-PRIVATE-KEY-DONT-SHARE";
 
-const RPC_API_KEY = "YOUR-API-KEY-FROM-INFURA-OR-ALCHEMY";
+const RPC_API_KEY = "YOUR-API-KEY-FROM-YOUR-RPC-PROVIDER";
```

Comment:
Here I blieve that you don't use the API_KEY rather you use the actual RPC_URL as said above in the JSON object:

```
sepolia: {
   url: RPC_URL,
   ...
   }
```

Reply:
what should we change it to @Markkos89 ?

Reply:
`RPC_URL` as this is an url starting with `https://..`

Reply:
"YOUR-API-KEY-URL-FROM-YOUR-RPC-PROVIDER"?

Reply:
"YOUR-API-KEY-URL-FROM-RPC-PROVIDER"

Reply:
if it is the first time a learner is using these tools, they won't see them very easily - that is the reason that it is written with a lot of words - to easily attract their attention to it.

Reply:
it would be `YOUR-RPC-URL-FROM-YOUR-RPC-PROVIDER` because the string is the RPC URL